draft-ietf-moq-transport.md

```diff
@@ -966,12 +988,18 @@ When a relay receives an incoming ANNOUNCE for a given namespace, for
 each active upstream subscription that matches that namespace, it SHOULD send a
 SUBSCRIBE to the publisher that sent the ANNOUNCE.
 
-Object headers carry a short hop-by-hop `Track Alias` that maps to
-the Full Track Name (see {{message-subscribe-ok}}). Relays use the
-`Track Alias` of an incoming Object to identify its track and find
-the active subscribers for that track. Relays MUST forward Objects to
-matching subscribers in accordance to each subscription's priority, group order,
-and delivery timeout.
+If a relay receives a Publisher-Chosen Subscription ID in SUBSCRIBE_OK, it
+SHOULD assign this ID to downstream subscriptions for the same track. Since
```

Comment:
But why, though?

Reply:
> more text explaining why relays SHOULD preserve these IDs

Well, that would be a good start.

> you saying you don't want any normative requirement for relays with respect to the IDs?

I have no idea as to what we're trying to accomplish here, so I can't say either way.

Reply:
I changed this from SHOULD to MAY.  The purpose is for systems that have such identifiers to convey them end-to-end.